k8s/containers/seed-update/Makefile

```diff
@@ -0,0 +1,13 @@
+SHELL=/bin/bash
+buildversion = $(shell cat version)
+
+.PHONY: all
+all: build
+
+.PHONY: increment
+increment:
+	echo $$(( $(buildversion) + 1 )) > version
```

Comment:
This assumes that the build numbers will be maintained in the file rather than defined by Jenkins. That doesn't sound like a good idea. Jenkins provides the build number as an environment variable. Can't you use that instead of the file?